corehq/apps/sms/models.py

```diff
@@ -775,6 +777,67 @@ class Meta:
         app_label = "sms"
 
 
+class CommConnectCaseSQL(CommCareCaseSQL, CommCareMobileContactMixin):
```

Comment:
This looks almost identical to `CommConnectCase`. Would it not make sense to make this a proxy wrapper instead of a subclass?